docs/user/pings/baseline.md

```diff
@@ -1,25 +1,63 @@
 # The `baseline` ping
 
 ## Description
-
 This ping is intended to provide metrics that are managed by the library itself, and not explicitly
 set by the application or included in the application's `metrics.yaml` file.
 
 ## Scheduling
-
-The `baseline` ping is automatically sent when the application is moved to the [background](index.md#defining-background-state).
+The `baseline` ping is automatically sent when the application is moved to the [background](index.md#defining-background-state) and it includes
+the following fields:
```

Comment:
To me it reads a bit weird to end a paragraph in `following fields:` to follow it up with a new heading and a new paragraph.

Reply:
Yeah -- I think it best to divide the sentence in two and put the second part in the `Contents` section.